docs/Appendix-J.rst

```diff
@@ -1,102 +1,147 @@
+In addition, the POWER-Up software server needs access to the following repositories during the preparation phase;
 
-    user@new-host:~/cluster-genesis/playbooks$ ansible-playbook -i hosts container/set_mgmt_switch_config.yml
+-  Red Hat 'common', 'optional' and 'extras'
+-  Extra Packages for Enterprise Linux (EPEL)
+-  Cuda Toolkit
+-  Anaconda
 
-Restore container from archive
-------------------------------
+These can be accessed using the public internet (URL's are provided) or via an alternate web site such as an intranet mirror repository or from a mounted USB key.
 
-#. Copy LXC file archive from origin to new host
+Before beginning, extract the contents of the powerai-enterprise-1.1.0_ppc64le.bin file and accept the license by running::
 
-#. Run 'container_restore.sh' script to install and start container::
+    $ sudo bash ./powerai-enterprise-1.1.0_ppc64le.bin
 
-    user@new-host:cluster-genesis/scripts $ ./container_restore.sh container_archive [new_container_name]
+NOTE: Extraction and license acceptance must be run on an OpenPOWER node. If you are running the POWER-Up installer software on an x_86 node, you must first extract the files on an OpenPOWER node and then copy all of the extracted contents to the POWER-Up installer node.
 
-#. Use LXC status to verify container is running::
+Preparation is run with the following POWER-Up command::
 
-    user@new-host:~$ sudo lxc-ls -f
+    $ pup software --prep paie52
+
+Preparation is interactive. Respond to the prompts as appropriate for your environment. Note that the EPEL, Cuda, dependencies and Anaconda repositories can be replicated from the public web sites or from alternate sites accessible on your intranet environment or from local disk (ie from a mounted USB drive). Most other files come from the local file system except for the Anaconda package which can be downloaded from the public internet during the preparation step.
+
+**Dependent software packages**
+The PowerAI Enterprise software is dependent on additional open source software that is not shipped with PowerAI Enterprise.
+These dependent packages are downloaded to the POWER-Up software server from enabled yum repositories during the preparation phase and are subsequently available to the client nodes during the install phase.
+Additional software packages can be installed in the 'dependencies' repo on the POWER-Up software server by listing them in the power-up/software/dependent-packages.list file.
+Entries in this file can be delimited by spaces or commas and can appear on multiple lines.
+Note that packages listed in the dependent-packages.list file are not automatically installed on client nodes unless needed by the PowerAI software.
+They can be installed on a client node explicitly using yum on the client node (ie yum install pkg-name). Alternatively, they can be installed on all client nodes at once using Ansible::
+
+    $ ansible all -i software_hosts -m yum -a "name=pkg-name"
```

Comment:
This command assumes the user will be in the 'playbooks/' directory (for the `-i software_hosts` path to work). I've been wondering if we should move it into the 'software/' directory with all of the other pai52 module code. In any case we should probably mention that this is a relative path dependent on the CWD.

Reply:
I think I like the idea of moving software_hosts to the software/ directory.